### PR TITLE
Add yubico-webauthn 2.4.0-RC2.wso2v1

### PR DESCRIPTION
**Pull request by ChanikaRuchini**

### CODE REVIEW

File: yubico-webauthn/2.3.0.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```
Comment (ThaminduDilshan) [inline yubico-webauthn/2.3.0.wso2v1/pom.xml:3]:
```suggestion
  ~ Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
```

File: yubico-webauthn/2.3.0.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+                    <groupId>com.google.guava</groupId>
+                    <artifactId>guava</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>com.upokecenter</groupId>
+            <artifactId>cbor</artifactId>
+            <version>${com.upokecenter.cbor.version}</version>
+        </dependency>
```
Comment (ThaminduDilshan) [inline yubico-webauthn/2.3.0.wso2v1/pom.xml:70]: Is this a approved dependency?

Comment (ThaminduDilshan) [reply yubico-webauthn/2.3.0.wso2v1/pom.xml:70]: nvm

File: yubico-webauthn/2.3.0.wso2v1/pom.xml (hunk 1 above)
Comment (ChanikaRuchini) [reply yubico-webauthn/2.3.0.wso2v1/pom.xml]: Fixed

**Pull request merged**